### ISSUE TITLE
The role=menu children should have a role=menuitem

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#5953]
When using the ARIA `role="menu"`, the items in the menu should have a `role="menuitem"`. This is important not only for semantics but also for interaction and the way assistive technologies announce the items.

Seems the items in the "More Options" menu (on the left) miss this attribute as opposed, for example, to the items in the block switcher menu (on the right):

<img width="569" alt="screen shot 2018-04-03 at 15 29 21" src="https://user-images.githubusercontent.com/1682452/38254206-f8886180-3758-11e8-96e0-753c3aeb57ac.png">

For completeness, menus can only contain `menuitems` and, optionally, a "separator", nothing else.
https://www.w3.org/TR/wai-aria-1.1/#menu
https://www.w3.org/TR/wai-aria-1.1/#menuitem
https://www.w3.org/TR/wai-aria-1.1/#separator

I'm not so concerned about the "titles" (e.g. "Transform into:") within the menus, they can be addressed in future iterations, if necessary.
